doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -102,4 +102,8 @@ In addition, we will start with a description of interfaces and capabilities (th
 
 As Kubernetes evolves over time it will add new technologies and capabilities and so the CNTT Reference Architecture will need to evolve with it. The final release cadence for the Reference Model changes and feature updates is still under discussion but the cadence of the Kubernetes Reference Architecture should at least follow the release cadence for Kubernetes, which is every three months and at least the support cadence, which is nine months (latest release, plus two previous releases).
 
+<p align="center"><img src="../figures/k8s_ref_arch_roadmap.png" alt="Kubernetes Reference Architecture Roadmap" title="Kubernetes Reference Architecture Roadmap" width="100%"/></p>
```

Comment:
Why the "VM-s orchestrated by Kubernetes" part is highlihted on the 2020 July release?

Reply:
I listed it because it is a fairly large change / addition.  However, I'm not dead set on having it in there (and we should review / edit after Prague anyway).

Reply:
@CsatariGergely I've updated as per the feedback in the call last week, are you ok with it now to resolve this conversation?

Reply:
The .ppt seems to be okay, but the .png is still the original one...

Reply:
Ah woops - have updated now, thanks.